### ISSUE TITLE
Update for Nextcloud 26.0

### ISSUE THREAD

**Issue by derEremit** [storj-thirdparty/nextcloud-app#35]
Hi. this is currently the only unsupported module thats keeping me from upgrading to nextcloud 26.

Much appreciated if this is just a simple version comatibility bump

**Comment by EchedelleLR** [comment]
I am starting to wonder if the project is abandoned at all...

**Comment by CodeShakingSheep** [comment]
Just fyi: you can also setup Storj as external storage in Nextcloud using "Amazon S3" and setting the hostname to "gateway.storjshare.io". That's how I did it for Nextcloud 26 and it works for me.

**Comment by EchedelleLR** [comment]
@CodeShakingSheep 

I ran into this issue due to faulty documentation:
https://github.com/nextcloud/server/issues/36427

Currently, the setup I am using is with the Nextcloud App in override mode to avoid Nextcloud disabling it. Sadly, there is no migration path/step between this ObjectStore and the S3-native one provided by Nextcloud. StorJ Nextcloud App saves files in a very different way than the native one.

In conclusion, it is impossible, as far as I am aware, to use the native S3 option for people using the StorJ Nextcloud App.

**Comment by EchedelleLR** [comment]
That said, I have no issues sponsoring this project if possible, so it is kept alive and could be used to support/promote StorJ specific features which are not possible with the AWS S3 SDK used by Nextcloud in its S3-native ObjectStore.

**Comment by jdavid82** [comment]
wait, @EchedeyLR are there no plans for fixing AWS S3 integration?

**Comment by jdavid82** [comment]
never mind I found the pull request https://github.com/nextcloud/documentation/pull/10437/files

**Comment by EchedelleLR** [comment]
What you do mean @jdavid82 ?

**Comment by jdavid82** [comment]
@EchedeyLR I just posted a comment in this PR: https://github.com/nextcloud/server/pull/39017#issuecomment-1646582944
I am not sure if you and I are talking about the same issue, but if we are then hopefully they can address it soon

**Comment by EchedelleLR** [comment]
Just one thing @jdavid82 now that I have some minutes: did you test migrating from StorJ to direct S3 integration (I mean same bucket just changing ObjectStore config) and got it working?

**Comment by jdavid82** [comment]
Hi @EchedeyLR , there must have been an issue with S3 at the time because `I didn't change anything` and a couple of days later it just started to work again

**Comment by jdavid82** [comment]
oh and I never had integration with Storj, but it might be cheaper than S3, I'll look at that in the future
